packages/zen-bull-netting/src/ZenBullNetting.sol

```diff
@@ -787,32 +723,23 @@ contract ZenBullNetting is Ownable, EIP712, FlashSwap {
             _params.wethUsdcPoolFee,
             usdcToRepay,
             _params.maxWethForUsdc,
-            uint8(FLASH_SOURCE.WITHDRAW_AUCTION),
-            abi.encodePacked(_params.withdrawsToProcess, oSqthAmount, usdcToRepay)
+            0,
+            abi.encodePacked(_params.withdrawsToProcess)
         );
 
         // send WETH to market makers
         IWETH(weth).deposit{value: address(this).balance - initialEthBalance}();
         toExchange = oSqthAmount;
         uint256 oSqthQuantity;
         for (uint256 i = 0; i < _params.orders.length && toExchange > 0; i++) {
-            if (_params.orders[i].quantity < toExchange) {
-                oSqthQuantity = _params.orders[i].quantity;
-            } else {
-                oSqthQuantity = toExchange;
-            }
-
-            IERC20(weth).transfer(
-                _params.orders[i].trader, (oSqthQuantity * _params.clearingPrice) / 1e18
-            );
-            toExchange -= oSqthQuantity;
-
-            emit WithdrawAuction(
+            toExchange = NettingLib.transferWethToMarketMaker(
```

Comment:
I don't think this function works.  ```        uint256 oSqthQuantity;``` is not used

Doesn't it always send 0 for oSQTHQuantity (which is supposed to be the MM order size)?

Reply:
fixed and added assert in test